chain/cmd/provider/main.go

```diff
@@ -20,7 +22,10 @@ import (
 	"github.com/bandprotocol/d3n/chain/x/zoracle"
 )
 
+const LIMIT_TIMEOUT = 3 * time.Second
```

Comment:
In go convention, private constant use lowerCamelCase. Therefore changed it to `limitTimeOut`